include/netpacket/if_addr.h

```diff
@@ -98,6 +98,7 @@ enum
   IFA_CACHEINFO,
   IFA_MULTICAST,
   IFA_FLAGS,
+  IFA_RT_PRIORITY,  /* u32, priority/metric for prefix route */
```

Comment:
Maybe we should include the description in comment block above instead of placing it here?